### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by ellismg**
Move to newer CoreFX

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -61,11 +61,15 @@
...
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
-    <CoreFxExpectedPrerelease>rc3-24128-00</CoreFxExpectedPrerelease>
+    <CoreFxExpectedPrerelease>rc3-24210-10</CoreFxExpectedPrerelease>
+    <ExternalExpectedPrerelease>rc3-24210-00</ExternalExpectedPrerelease>
   </PropertyGroup>
 
   <ItemGroup>
-    <ValidationPattern Include="^(?i)((System\..%2A)|(Microsoft\.Cci)|(Microsoft\.CSharp)|(Microsoft\.Win32\..%2A)|(Microsoft\.VisualBasic))(?&lt;!TestData)$">
+    <ValidationPattern Include="^(?i)((System\..%2A)|(Microsoft\.CSharp)|(Microsoft\.Win32\..%2A)|(Microsoft\.VisualBasic))(?&lt;!TestData)$">
```
Comment (ericstj) [inline dir.props]: Need to add NETStandard.Library here.

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json [hunk 2]
```diff
@@ -11,13 +11,13 @@
         "Microsoft.Build": "0.1.0-preview-00022",
         "Microsoft.Net.Compilers.NetCore": "1.3.0-beta1-20160429-01",
         "Microsoft.Net.Compilers.Targets.NetCore": "0.1.4-dev",
-        "Microsoft.Cci": "4.0.0-rc3-24128-00",
+        "Microsoft.Cci": "4.0.0-rc3-24210-10",
         "Microsoft.Composition": "1.0.30",
         "Microsoft.NETCore.App": "1.0.0-rc3-002857",
         "Microsoft.NETCore.TestHost": "1.0.0-rc3-24123-01",
         "NETStandard.Library": "1.6.0-rc3-24208-04",
```
Comment (ericstj) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json]: This guy is the problem.

File: dir.props [hunk 3]
```diff
@@ -61,11 +61,15 @@
...
+      <ExpectedPrerelease>$(CoreFxExpectedPrerelease)</ExpectedPrerelease>
+    </ValidationPattern>
+    <ValidationPattern Include="^Microsoft.Cci$">
       <ExpectedPrerelease>$(CoreFxExpectedPrerelease)</ExpectedPrerelease>
```
Comment (ericstj) [inline dir.props]: This needs to be ExternalExpectedPrerelease

Comment (ellismg) [inline dir.props]: Ugh, yup.

**Pull request merged**